transformers/component_tests/queries/test_helpers/test_helpers.go

```diff
@@ -235,6 +235,26 @@ func CreateIlk(db *postgres.DB, header core.Header, valuesMap map[string]interfa
 	CreateCatRecords(db, header, valuesMap, catMetadatas, catRepo)
 	CreateJugRecords(db, header, valuesMap, jugMetadatas, jugRepo)
 	CreateSpotRecords(db, header, valuesMap, spotMetadatas, spotRepo)
+	if len(vatMetadatas) >= 1 {
```

Comment:
What's the rationale behind this predicate? Are there places where we want to create metadata for non-vat contracts but not for the vat?

Reply:
Yeah, we're doing that a few times in the tests for `get_ilk`. E.g. https://github.com/makerdao/vdb-mcd-transformers/blob/vdb-1064-replace-ilk-query/transformers/component_tests/queries/get_ilk_query_test.go#L233

So I guess we could get rid of this conditional when we delete the `get_ilk` function, after the `ilk_state` type is deleted.

Reply:
Awesome. My initial thought is to wonder whether those tests corrrectly simulating realistic conditions, but I suppose it probably makes sense that we'd want to be able to verify that the records are correct even if some important data is missing.